packages/dapp/src/types/internal.types.ts

```diff
@@ -79,87 +71,6 @@ export interface ChainsMenuListItem {
   showButton?: boolean;
 }
 
-export interface SwapPageStartParams {
-  depositChain?: ChainKey;
-  depositToken?: string;
-  depositAmount: BigNumber;
-  withdrawChain?: ChainKey;
-  withdrawToken?: string;
-}
-
-export interface Amounts {
-  amount_coin: BigNumber;
-  amount_usd: BigNumber;
-}
-
-export interface DataType {
-  [key: string]: string | number | Amounts | Coin; // kind of deactivating typing for DataType; last resort?
-  key: React.Key;
-  coin: Coin;
-  portfolio: Amounts;
-}
-
-export function chainKeysToObject(val: any) {
-  const result: { [ChainKey: string]: any } = {};
-  for (const key in ChainKey) {
-    result[key.toLowerCase()] = JSON.parse(JSON.stringify(val));
-  }
-  return result;
-}
-
-export interface Wallet {
-  address: string;
-  loading: boolean;
-  portfolio: { [ChainKey: string]: Array<TokenAmount> };
-}
-
-export enum Currencies {
-  USD = 'usd',
-  EUR = 'eur',
-}
-
-export interface SummaryAmounts {
-  amount_usd: BigNumber;
-  percentage_of_portfolio: BigNumber;
-}
-
-export interface WalletSummary {
-  wallet: string;
-  chains: {
-    [ChainKey: string]: SummaryAmounts;
-  };
-}
-
-export interface WalletConnectInfo {
-  accounts: string[];
-  bridge: string;
-  chainId: number;
-  clientId: string;
-  clientMeta: { [k: string]: any }; // not important as of now
-  connected: boolean;
-  handshakeId: number;
-  handshakeTopic: string;
-  key: string;
-  peerId: string;
-  peerMeta: { [k: string]: any }; // not important as of now
-}
-
-export interface WalletContextProps {
-  account: WalletAccount;
-  addChain(chainId: number): Promise<boolean>;
-  addToken(chainId: number, token: Token): Promise<void>;
-  disconnect(): void;
-  switchChain(chainId: number): Promise<boolean>;
-  connect(wallet?: WalletManagementWallet | undefined): Promise<void>;
-}
-
-export interface WalletAccount {
-  address?: string;
-  isActive?: boolean;
-  signer?: Signer;
-  chainId?: number;
-}
-
 export type MultisigWidgetConfig = Pick<
   WidgetConfig,
   'fromChain' | 'requiredUI'
```

Comment:
All of that was junk not in use. 
Wallet related stuff was duplicated and should be used from wallet.ts instead, so got rid of what was found in here.